rules/cross-platform/external_alerts.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2020/07/08"
+ecs_version = ["1.5.0"]
+maturity = "development"
+updated_date = "2020/07/08"
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert each time an external alert is received. Enabling this rule allows you to immediately being
+investigating external alerts in the app.
+"""
+language = "kuery"
+license = "Elastic License"
+name = "External Alerts"
```

Comment:
is this displayed/used anywhere in the UI when a `rule_name_override` is defined?

Reply:
Yeah, this'll be used within the `Rules Table` and on the `Rule Details` page as well. When defined, `rule_name_override` is only used to populate `signal.rule.name`, so mostly specific to visualizing signals.